src/meltano/cli/__init__.py

```diff
@@ -4,8 +4,10 @@
 import logging
 import os
 import sys
+from textwrap import dedent
 from typing import TYPE_CHECKING
```

Comment:
This suggestion pairs with the one that adds a type signature to `handle_meltano_error` below.

```suggestion
from typing import TYPE_CHECKING, NoReturn
```